### PR TITLE
Move CONTRIBUTING.md + Add Test for Adding New Models

### PR DESCRIPTION
**Pull request by NirantK**

### CODE REVIEW

File: CONTRIBUTING.md [hunk 1]
```diff
@@ -63,10 +63,11 @@ Even documentation improvements and tests are most welcome:
...
 
 ### Adding New Models
-You can start by adding new models to the FastEmbed. You can find all the model requests [here](https://github.com/qdrant/fastembed/labels/model%20request). 
-
-
-There are quite a few pull requests that were merged for this purpose and you can use them as a reference. Here is an example: https://github.com/qdrant/fastembed/pull/129 
+1. Open Requests for New Models are [here](https://github.com/qdrant/fastembed/labels/model%20request). 
+2. There are quite a few pull requests that were merged for this purpose and you can use them as a reference. Here is an example: https://github.com/qdrant/fastembed/pull/129
+3. Make sure to add tests for the new model
+    - The CANONICAL_VECTOR values must come from a reference implementation usually from Huggingface Transformers or Sentence Transformers
```
Comment (KShivendu) [inline CONTRIBUTING.md:69]: Please describe what's canonical vector. 

Do you mean one has to convert these two 
```
input_texts = [
    "hello world", "flag embedding"
]
```

Using HF Inference API for [the model](https://huggingface.co/intfloat/multilingual-e5-small)?

**Review by KShivendu — COMMENTED** [review]
The part of checking in notebook is a bit unusual. Ideally it could be a Github workflow triggered with a button. But I think it's okay since model merges won't be that frequent.

**Comment by KShivendu** [comment]
Ohh wait we already have a test for [this](https://github.com/qdrant/fastembed/blob/main/tests/test_text_onnx_embeddings.py#L32-L48). Is there any blocker from running it in CI?

File: CONTRIBUTING.md (hunk 1 above)
Comment (NirantK) [reply CONTRIBUTING.md:69]: Reference PRs are linked before that sentence

**Comment by NirantK** [comment]
@KShivendu that test also depends on the CANONICAL_VECTOR_VALUES — that's the ground truth, and contributors [mistake test for the implementation](https://github.com/qdrant/fastembed/pull/153)

Hence, that needs to be outside of this repository ideally — and I don't want to clutter this repository and make the tests slower than they already are

**Pull request merged**